packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -188,6 +195,14 @@ contract FederatedAttestations is
     }
   }
 
+  modifier isValidUser(address issuer, address account, address signer) {
+    require(
+      msg.sender == account || msg.sender == issuer || msg.sender == signer,
+      "User does not have permission to perform this action"
+    );
+    _;
+  }
+
```

Comment:
Is this necessary? Should we allow anyone who has possession of a valid signature to register the attestation? Since it isn't possible to revoke individual attestations, is there a risk of someone somehow gaining access to a signature and registering it onchain when the issuer/account doesn't want to have it onchain?